pip/wheel.py

```diff
@@ -37,6 +37,12 @@
 from pip.utils.temp_dir import TempDirectory
 from pip.utils.ui import open_spinner
 
+try:
+    from concurrent.futures.process import ProcessPoolExecutor
```

Comment:
The vendored version could fail on unreleased newer python versions. Trying to maintain python core libraries is just duplicating efforts.